### PR TITLE
W-13118963: Generate a module-info.java for mule bootstrap and logging

### PR DESCRIPTION
**Pull request by elrodro83**
modules

### CODE REVIEW

**Comment by elrodro83** [comment]
--validate

**Comment by elrodro83** [comment]
--validate

File: modules/reboot/src/main/java/org/mule/runtime/module/reboot/api/MuleContainerBootstrapUtils.java [hunk 1]
```diff
@@ -81,6 +85,7 @@ public static File getMuleAppsDir() {
     return isStandalone() ? new File(getMuleBase(), MULE_APPS_FILENAME) : null;
   }
 
+  // TODO remove
```
Comment (eze210) [inline modules/reboot/src/main/java/org/mule/runtime/module/reboot/api/MuleContainerBootstrapUtils.java:88]: Missing ticket? Or missing in this PR?

**Review by eze210 — CHANGES_REQUESTED** [review]

**Review by eze210 — DISMISSED** [review]

**Comment by elrodro83** [comment]
--validate

**Comment by elrodro83** [comment]
--validate

**Review by eze210 — APPROVED** [review]

**Comment by elrodro83** [comment]
--validate

**Pull request merged**